src/main/io/gps.c

```diff
@@ -120,6 +120,12 @@ static gpsProviderDescriptor_t  gpsProviders[GPS_PROVIDER_COUNT] = {
     { GPS_TYPE_NA, 0, false,  NULL, NULL },
 #endif
 
+    /* MTK GPS */
+#ifdef GPS_PROTO_MTK
+    { GPS_TYPE_SERIAL, MODE_RXTX, false, NULL, &gpsHandleNMEA },
```

Comment:
I this we should move this to a separate function gpsHandleMTK in a separate module to avoid confusion.